exercises/diffie-hellman/README.md

```diff
@@ -0,0 +1,69 @@
+# Diffie-Hellman
+
+Diffie-Hellman key exchange.
+
+Alice and Bob use Diffie-Hellman key exchange to share secrets.  They
```

Comment:
Would it be more informative to place a trusted long-living reference to a detaild description here? 

Like [Diffie-Hellman key exchange](https://en.wikipedia.org/wiki/Diffie%E2%80%93Hellman_key_exchange)